src/man/sssd-kcm.8.xml

```diff
@@ -162,6 +162,42 @@ systemctl restart sssd-kcm.service
         </para>
     </refsect1>
 
+    <refsect1 id='renewals'>
+        <title>RENEWALS</title>
+        <para>
+            The sssd-kcm service can be configured to attempt TGT
+            renewal for renewable TGTs stored in the KCM ccache.
+            Renewals are only attempted when half of the ticket
+            lifetime has been reached. KCM Renewals are configured
+            when the following options are set in the [kcm] section:
+            <programlisting>
+tgt_renewal = true
+krb5_renew_interval = 60m
+            </programlisting>
+        </para>
+        <para>
+            SSSD can also inherit krb5 options for renewals from an existing
+            domain.
+        </para>
+            <programlisting>
+tgt_renewal = true
+tgt_renewal_inherit = testad.domain
```

Comment:
use generic "domain-name" instead of testad.domain

Reply:
Fixed.